### DISCUSSION TITLE
Discussion for layouts and colors

### DISCUSSION THREAD

**Discussion by jdwilkin4** [jdwilkin4/contrast-ratio-repo#10] [Ideas]
This discussion should only be concerning the general layout and colors we want to use for this project.
This addresses #3

**Comment by AndreyDmitriev39r** [comment]
So, my main thought about layout:
We should stick to something very simple for colors. Maybe even just black text / white background.
Because user will come to figure out stuff about their own color schemes. Last thing they wanna do - to be distracted by our own color scheme.
Maybe it's just me - I am content-oriented guy, I don't care about design much as a user.

Responsiveness: it's not really my thought, I guess it's common practice: I guess for the start we should be concerned with layout for small screens(mobile first). Then create breakpoints for bigger viewports

My vision of basic layout can be gathered from the link below - it's codepen. I guess anybody who wants to modify my pen, add their own vision, can fork it there(tho I never forked anything on codepen).

I used text content of the page and CSS comments both to express my thoughts.

I tried to use Figma first for mockup, but... there are like too many different buttons. Need some time to get familiar with it.

https://codepen.io/AndreyDmitriev39r/pen/eYMrQwj

**Reply by jdwilkin4** [reply]
@AndreyDmitriev39r 
> We should stick to something very simple for colors.

That works for me.
We should definitely have the red and green colors to show good and bad contrasts.

> Responsiveness: it's not really my thought, I guess it's common practice:

It should be responsive.
I am fine with a mobile first approach. 

As for your layout, I don't think we need a navbar
For your block 2 section, we can just have a title (h1)

I am fine with the placement of the contrast checker and the short explanation afterwards.
It's similar to what they have in the WenAIM page 
https://webaim.org/resources/contrastchecker/

For the footer, maybe we can place a copyright for this app with the year.

**Comment by jdwilkin4** [comment]
I am going to link @Kernix13 's codepen here as part of this ongoing discussion. 
https://codepen.io/jim-kernicky/pen/YzaLRVg?editors=1100

**Comment by Okibe00** [comment]
@Kernix13 layout looks simple and easy enough to use, I think we should go with it.

**Reply by AndreyDmitriev39r** [reply]
Actually, this version of basic layout already implemented and merged. Check out codebase. And make sure to pull the changes to your local repo.

**Comment by Kernix13** [comment]
I just spun up something basic based on the Figma design I made. I took a look at all the links that I posted in Discord and "borrowed" designs that I liked. I'm not happy that I used so many flex containers, but I'm better at flex than with grid, plus its 2 rows with different elements and design. I think it could be way better but I wanted someone to work on it who wants some CSS pushes.

**Reply by AndreyDmitriev39r** [reply]
I think you did good job for the start. We can always add more styling or refactor existing one, for now we have general layout and that's easy to move on with this
